aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -87,15 +87,22 @@ At the end of this tutorial, you'll have a working Razor Pages web app that you'
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-From a terminal, run the following command:
+* Select **File** > **New Solution**.
 
-<!-- TODO: update these instruction once mac support 2.2 projects -->
+![macOS New solution](../first-mvc-app/start-mvc/_static/new_project_vsmac.png)
 
-```console
-dotnet new webapp -o RazorPagesMovie
-```
+* Select **.NET Core** > **App** > **Web Application** > **Next**.
+
+  ![macOS New project dialog](razor-pages-start/_static/webapp.png)
+
+* In the **Configure your new ASP.NET Core Web API** dialog, set the  **Target Framework** of **.NET Core 2.0**.
```

Comment:
In the **Configure your new ASP.NET Core Web API** dialog, set the  **Target Framework** to **.NET Core 2.2**.

Reply:
You want 2.2, not 2.0.

Reply:
My bad there, thanks @Rick-Anderson